src/vhip/mlmodel/gene_features.py

```diff
@@ -9,7 +9,7 @@
 import os
 from typing import List
 
-from .read_sequence import read_annotated_genes
+from vhip.mlmodel.read_sequence import read_annotated_genes
 
```

Comment:
is there a reason why you switch from relative import to absolute?

Reply:
Right. I noticed that sometimes the prefix was there and sometimes not. I have no idea, but the relative stopped working yesterday. Let me try to put it back

Reply:
We should be consistent and do relative import every time!